src/BenchmarkDotNet.Core/Exporters/MarkdownExporter.cs

```diff
@@ -160,5 +160,11 @@ private void PrintTable(SummaryTable table, ILogger logger)
                 rowCounter++;
             }
         }
+
+        private static string getDelimiterLineForColumn(SummaryTable.SummaryTableColumn column)
+        {
+            var justifier = column.Justify == SummaryTable.SummaryTableColumn.TextJustification.Left ? " " : ":";
```

Comment:
Could we use a `switch` statement here? With the `? :` operator, it would be tricky to find a bug when we decided to add another value in the enum (e.g. `Center`).